doc/cygnus-ngsi/installation_and_administration_guide/ipv6_support.md

```diff
@@ -0,0 +1,37 @@
+#<a name="top"></a>IPv6 support
+Content:
+
+* [Service endpoint](#section1)
+* [API](#section2)
+* [GUI](#section3)
+
+##<a name="section1"></a>Service endpoint 
+Native Flume Http sources supprt IPv6, therefore cygnus-ngsi supports IPv6 at its service endpoint.
+
+It is just a matter of configuring the Http source `bind` parameter (which by default takes the value `127.0.0.1` when not explicitely configured) to `::` (undefined address) or `::1` (IPv6 localhost).
+
+For instance, let's supose a cygnus-ngsi focused on persisting NGSI data in MySQL. IPv6 is enabled in the Http source as said above:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1260/commits/cfe809b0b4d83351cfa3a1182d331dfac8466c7a